<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>351</width>
      <height>588</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButtonPut">
             <property name="text">
              <string>Put</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Tamanho Da Matriz</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tamanhoDaMatrizX">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tamanhoDaMatrizY"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tamanhoDaMatrizZ"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnTamanhoDaMtariz">
             <property name="text">
              <string>Confirma</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonBox">
           <property name="text">
            <string>Box</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="boxX"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="boxY"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="boxZ"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnPutBox">
           <property name="text">
            <string>Put</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCutBox">
           <property name="text">
            <string>Cut</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButtonEllipsoid">
             <property name="text">
              <string>Ellipsoid</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="elipsoidX"/>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="elipsoidY"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="elipsoidZ"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnPutElipsoid">
             <property name="text">
              <string>Put</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCutElipsoid">
             <property name="text">
              <string>Cut</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="radioButtonSphere">
               <property name="text">
                <string>Sphere</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Raio</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="raioSphere"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnPutSphere">
               <property name="text">
                <string>Put</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCutSphere">
               <property name="text">
                <string>Cut</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Muda Cor</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>R</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mudaCorR"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string>G</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mudaCorG"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>B</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mudaCorB"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>A</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mudaCorA"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="btnMudaCor">
                 <property name="text">
                  <string>Confirma</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Seção Z</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber"/>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSliderZTela">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Plotter" name="plotter" native="true">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>9</y>
      <width>331</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderZTela</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
